cmd/provider_cmd_squadcast.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"log"
+
+	squadcast_terraforming "github.com/GoogleCloudPlatform/terraformer/providers/squadcast"
+
+	"github.com/GoogleCloudPlatform/terraformer/terraformutils"
+	"github.com/spf13/cobra"
+)
+
+func newCmdSquadcastImporter(options ImportOptions) *cobra.Command {
+	var refreshToken string
+	var teamName string
+
+	cmd := &cobra.Command{
+		Use:   "squadcast",
+		Short: "Import current state to Terraform configuration from Squadcast",
+		Long:  "Import current state to Terraform configuration from Squadcast",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			originalPathPattern := options.PathPattern
+			for _, region := range options.Regions {
+				provider := newSquadcastProvider()
+				options.PathPattern = originalPathPattern
+				options.PathPattern += region + "/"
+				log.Println(provider.GetName() + " importing region " + region)
```

Comment:
mostly not required

Reply:
Specifying PathPattern is necessary for creating specific folder (us/ or eu/) in the generated folder

Reply:
ok